src/ensemble/GrandCanonical.cpp

```diff
@@ -556,7 +556,6 @@ void ChemicalPotential::grandcanonicalStep(
 //			moleculeContainer->_cells[cellid].addParticle(m);
 			DeltaUpot = moleculeContainer->getEnergy(&particlePairsHandler, mit,
 					*cellProcessor);
-			domain->submitDU(this->getComponentID(), DeltaUpot, ins);
```

Comment:
This is worrysome. Don't delete the function call. You can comment the code inside of the function call out, but don't delete it, so that we remember that there was something there.

Reply:
changed as requested 6027c17